packages/core/src/controllers/relayer.ts

```diff
@@ -304,6 +335,41 @@ export class Relayer extends IRelayer {
   }
 
   // ---------- Private ----------------------------------------------- //
+  /*
+   * In Node, we must detect when the connection is stalled and terminate it.
+   * The logic is, if we don't receive ping from the relay within a certain time, we terminate the connection.
+   * The timer is refreshed on every message received from the relay.
+   *
+   * In the browser, ping/pong events are not exposed, so the above behaviour is handled by `subscribeToNetworkChange` and `isOnline` functions.
+   */
+  private startPingTimeout() {
+    if (!isNode()) return;
+    try {
+      //@ts-ignore
```

Comment:
`expect-error` generally the better one to use if really needed, since TS will flag if there is no longer a type error here.

(guessing this is why, please adjust as needed)
```suggestion
      //@ts-expect-error - Types are divergent between the node and browser WS API
```